### PR TITLE
READMEs for all samples

### PR DESCRIPTION
**Pull request by JAG-UK**

### CODE REVIEW

**Review by mmwilder26 — COMMENTED** [review]
For door entry readme.md . . 
Line 3:  "dor" --> "door" referencing below. ..
connected dor locks it's a real world issue

Line 5:  "wold" --> "world" referencing below. . .
By combining virtual wold evidence (cryptographic identities, timestamps)

File: archivist_samples/signed_records/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Signed Records sample
+
+A key aspect of Zero Trust architectures is the removal of inherent trust in any individual part of your network. The best way to assure trustworthy operations is to combine several best-of-breed security technologies and then adopt a trust-but-verify approach to all of them.
+
+One such strong technology is to use device certificates or cryptographic keys on your IoT devices and use them to authenticate messages, proving that the message came from a particular piece of hardware. This of course is not foolproof: a compromised device with a strong key can send some very secure bad messages! But it is a strong defence against simple network-borne attacks like spoofing and man-in-the-middle forgeries.
+
+RKVST already supports high integrity authentication from secure devices trough its flexible command authorization scheme (leading to the 'who' in 'when who did what') but this can be added to by also signing the message contents (the 'what') with a device key. This signature is then verifiable independent of RKVST and provides additional proof of authenticity when it's needed.
```
Comment (mmwilder26) [inline archivist_samples/signed_records/README.md:7]: trough --> through referencing below. . . 
secure devices trough its flexible command

Comment (JAG-UK) [reply archivist_samples/signed_records/README.md:7]: Addressed locally. Will push when all comments are collected.

Comment (creator-of-data-assurance-hub) [reply archivist_samples/signed_records/README.md:7]: "who did what when" is the order of things now.

File: archivist_samples/door_entry/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Door entry sample
+
+Access control is not just an issue for files and computer systems: when it comes to connected dor locks it's a real world issue. Various authorities (landlords, building services, police, delivery companies) have legitimate reasons to enter the communal areas of shared buildings but the residents and owners of that building also need to be sure of their safety and privacy. How do you makes sure that privileged access to buildings is enabled whilst also holding the authorities to account and preventing abuses?
```
Comment (creator-of-data-assurance-hub) [inline archivist_samples/door_entry/README.md:3]: various "stakeholders" rather than authorities.

File: archivist_samples/door_entry/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+RKVST Data Assurance Hub offers a solution to this problem: by ensuring that all stakeholders have transparent access to records of use of privilege, abuses are discouraged and quickly discovered. By combining virtual wold evidence (cryptographic identities, timestamps) with real-world evidence (photographs from the built-in camera from the door access system) a very strong record of when who accessed a building is maintained and made available for audit and dispute resolution.
```
Comment (creator-of-data-assurance-hub) [inline archivist_samples/door_entry/README.md:5]: "have a transparent view of privileged access events"... "virtual world"

File: archivist_samples/signed_records/README.md (hunk 1 above)
Comment (JAG-UK) [reply archivist_samples/signed_records/README.md:7]: Order changed

File: archivist_samples/signed_records/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Signed Records sample
+
+A key aspect of Zero Trust architectures is the removal of inherent trust in any individual part of your network. The best way to assure trustworthy operations is to combine several best-of-breed security technologies and then adopt a trust-but-verify approach to all of them.
```
Comment (creator-of-data-assurance-hub) [inline archivist_samples/signed_records/README.md:3]: it's "verify THEN trust"

**Review by creator-of-data-assurance-hub — DISMISSED** [review]
changes please!

File: archivist_samples/signed_records/README.md (hunk 4 above)
Comment (JAG-UK) [reply archivist_samples/signed_records/README.md:3]: incorporated

File: archivist_samples/door_entry/README.md (hunk 3 above)
Comment (JAG-UK) [reply archivist_samples/door_entry/README.md:5]: Nice. done

File: archivist_samples/door_entry/README.md (hunk 2 above)
Comment (JAG-UK) [reply archivist_samples/door_entry/README.md:3]: Nice. Done

File: archivist_samples/door_entry/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Door entry sample
+
+Access control is not just an issue for files and computer systems: when it comes to connected door locks it's a real world issue. Various stakeholders (landlords, building services, police, delivery companies) have legitimate reasons to enter the communal areas of shared buildings but the residents and owners of that building also need to be sure of their safety and privacy. How do you makes sure that privileged access to buildings is enabled whilst also holding the authorities to account and preventing abuses?
+
+RKVST Data Assurance Hub offers a solution to this problem: by ensuring that all stakeholders have have a transparent view of privileged access events, abuses are discouraged and quickly discovered. By combining virtual world evidence (cryptographic identities, timestamps) with real world evidence (photographs from the built-in camera from the door access system) a very strong record of when who accessed a building is maintained and made available for audit and dispute resolution.
```
Comment (eccles) [inline archivist_samples/door_entry/README.md:5]: duplicate 'have'  stakeholders have have

**Review by eccles — DISMISSED** [review]

File: archivist_samples/door_entry/README.md (hunk 5 above)
Comment (JAG-UK) [reply archivist_samples/door_entry/README.md]: Done

**Review by eccles — APPROVED** [review]

**Pull request merged**